### PR TITLE
Fix tfs start failure due to null version number

### PR DESCRIPTION
**Pull request by satishpasumarthi**
*Issue #, if available:*
 If the model.tar.gz contains zero as the version number, the TFS Server fails to start due to null version number in model config protobuf file. Additionally, failures of such nature goes into multiple retry loops and restarts and takes ~30 mins+ to report the failure in starting the endpoint. 

*Description of changes:*
- Make`find_model_versions` function in the file `tfs_utils.py`, robust to handle scenarios when the model version number is `0`
- Added black-checks to the `tox` env
- Formatted files using `black`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/build_artifacts/sagemaker/tfs_utils.py [hunk 1]
```diff
@@ -135,7 +146,12 @@ def find_models():
 
 
 def find_model_versions(model_path):
-    return [version.lstrip("0") for version in os.listdir(model_path) if version.isnumeric()]
```
Comment (satishpasumarthi) [inline docker/build_artifacts/sagemaker/tfs_utils.py:138]: @josephevans This is the actual change for this PR

**Comment by satishpasumarthi** [comment]
@josephevans could you please review this PR?

**Review by satishpasumarthi — COMMENTED** [review]
This is the fix

**Review by josephevans — APPROVED** [review]
LGTM!

File: docker/build_artifacts/deep_learning_container.py [hunk 2]
```diff
@@ -75,8 +89,10 @@ def query_bucket():
     region = _retrieve_instance_region()
 
     if instance_id is not None and region is not None:
-        url = ("https://aws-deep-learning-containers-{0}.s3.{0}.amazonaws.com"
-               "/dlc-containers.txt?x-instance-id={1}".format(region, instance_id))
+        url = (
+            "https://aws-deep-learning-containers-{0}.s3.{0}.amazonaws.com"
```
Comment (saimidu) [inline docker/build_artifacts/deep_learning_container.py:93]: Do we need to continue to have this file on this repo? Can we remove it?

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,6 @@ __pycache__
 .tox/
 log.txt
 .idea/
+node_modules/
+package.json
+package-lock.json
```
Comment (saimidu) [inline .gitignore:7]: nit: Ending a file with a new line prevents the last line from appearing in future commits as a change.
```suggestion
package-lock.json

```

**Review by saimidu — APPROVED** [review]

File: docker/build_artifacts/deep_learning_container.py (hunk 2 above)
Comment (satishpasumarthi) [reply docker/build_artifacts/deep_learning_container.py:93]: We are planning a clean up of this repo which involves revisting/updating some tests. As of now this file is used while building a docker image (mimic DLC) and host tfs model server in some tests. We will keep it for now.

**Review by saimidu — APPROVED** [review]

**Pull request merged**